pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -543,6 +543,78 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 			},
 		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryBasicUserRoleName,
```

Comment:
I think this may fall out and just use our normal basic-user.  Let's remove it from this pull and see if we need to re-add it later.